osbs/build/build_request.py

```diff
@@ -732,6 +737,11 @@ def render(self, validate=True):
             self.spec.validate()
         super(ProductionBuild, self).render()
 
+        repo_name = git_repo_humanish_part_from_uri(self.spec.git_uri.value)
+        # What about forked repos?
```

Comment:
This relates to the build config name collision issue I mentioned above. Options are: pick a new name; error out.